tox.ini

```diff
@@ -59,7 +59,7 @@ commands =
     rm -f docs/modules.rst
     make -e -C docs clean
     make -e -C docs html
-    python setup.py check --restructuredtext
+    rstcheck -r .
```

Comment:
I'd love a tiny ADR about the motivation for moving from the standard check via setuptools to using the new tool.  Why switch? What problem were you facing that couldn't be solved with the standard tools on the paved road(cookiecutter).

Reply:
See more information in the linked PR here: https://github.com/openedx/edx-cookiecutters/pull/204#discussion_r899287280